apps/backend/src/main/java/no/nav/data/etterlevelse/etterlevelseDokumentasjon/domain/EtterlevelseDokumentasjon.java

```diff
@@ -96,6 +97,30 @@ public EtterlevelseDokumentasjonResponse toResponse() {
                 .build();
     }
 
+    public EtterlevelseDokumentasjonGraphQlResponse toGraphQlResponse() {
```

Comment:
Jeg hadde foretrukket å ha dette i EtterlevelseDokumentasjonGraphQlResponse. Da slipper domenemodellen å forholde seg til presentasjonslaget. Men ikke så farlig for nå (siden vi synder stort på dette området allerede).

Reply:
blir  ikke det litt lengre kode når vi konverterer?
får ikke brukt < .convert(EtterlevelseDokumentasjon::toGraphQlResponse) >